src/org/martinklepsch/derivatives.cljc

```diff
@@ -167,6 +168,23 @@
   [state drv-k]
   (rum/react (get-ref state drv-k)))
 
+(defn drvs [& ks]
+  (let [ds (map drv ks)
+        will-mount (rutil/collect :will-mount ds)
+        will-unmount (rutil/collect :will-unmount ds)]
+    {:class-properties (:class-properties (first ds))
```

Comment:
Oh, I'm not sure - the `d/drv` has the `:class-properties`, so I wanted to propagate it to make sure nothing breaks - could remove it if it's not needed, but wondering if the `get!` and `set!` will be accessible from the context without it?